packages/amplify-codegen/src/utils/getRelativeTypesPath.js

```diff
@@ -2,7 +2,11 @@ const path = require('path');
 
 function getRelativeTypesPath(opsGenDirectory, generatedFileName) {
   if (generatedFileName) {
-    const relativePath = path.relative(opsGenDirectory, generatedFileName);
+    const relativePath = path
+      .relative(opsGenDirectory, generatedFileName)
+      // ensure posix path separators are used
+      .split(path.win32.sep)
+      .join(path.posix.sep);
```

Comment:
I suspect this is OK. But, is there any possibility that this affects any *other* downstream consumers of the `relativePath` in unintended (negative) ways?

Reply:
All `path` and `fs` functions accept posix style paths even on a windows machine. So I think it should be fine.

Reply:
i.e.
```
> path.resolve('./packages/graphql-generator/package.json')
D:\\Users\\dppilche\\amplify-codegen\\packages\\graphql-generator\\package.json
```

Reply:
I'm more so wondering if we do any path splitting or merging that could break due to it looking for os delimiting.